### ISSUE TITLE
 Is the line 30 of encoder_channelatt_img.py inconsistent with your paper?

### ISSUE THREAD

**Issue by NOTGOOOOD** [ShengtianSang/AFMA#2]
Is the line 30 of encoder_channelatt_img.py inconsistent with your paper?
**In code**
`
self.conv_img = nn.Sequential(
            nn.Conv2d(3, 64, kernel_size=(7, 7), padding=3),
            nn.Conv2d(64, 12, kernel_size=(3, 3), padding=1)
        )
`
**In paper**
Step 1 . One 7×7 conv... with 64 filters and one 3×3 conv... with 1 filter ...

So, why
`nn.Conv2d(64, 12, kernel_size=(3, 3), padding=1)`
instead of
`nn.Conv2d(64, 1, kernel_size=(3, 3), padding=1)`

**Comment by ShengtianSang** [comment]
Hi, thank you for your question. Line 30 of encoder_channelatt_img.py is consistent with the paper. I think what confuses you is why it is 12 here. I used CamVid dataset as an example to train the model, and CamVid has 12 tag categories. I've updated the code to change this to the symbol num_classes, which indicates the number of categories in the dataset used.
